frontend/src/signup/components/forms/InputField.jsx

```diff
@@ -1,26 +1,16 @@
 import React from 'react';
 import FieldError from './FieldError';
 
-const InputField = ({ input, label, placeholder, type, optional, id, meta: { touched, error } }) => {
```

Comment:
Not necessarily a change, but this removes optional/mandatory handling from the component (we aren't doing anything with it at the moment though).